guides/spin/cli/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: single
+title:  "Install and Configure Spin CLI"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Install `spin`
+
+One way to manage applications and pipelines as code is through [spin](http://github.com/spinnaker/spin).
+
+To acquire `spin`, do the following:
+
+```bash
+# See the versions released at https://github.com/spinnaker/spin/releases.
+export VERSION=<desired spin version tag> # Select a tag from the released version, e.g. v0.1.0
```

Comment:
Where can I find available version tags? I don't see /spinnaker/spin/releases/ in the repo. Is that to be added when we're ready to release?